packages/block-library/src/comments-pagination/edit.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	InspectorControls,
+	useBlockProps,
+	useInnerBlocksProps,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
+import { useSelect } from '@wordpress/data';
+import { getBlockSupport } from '@wordpress/blocks';
+import { PanelBody } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { CommentsPaginationArrowControls } from './comments-pagination-arrow-controls';
+
+const TEMPLATE = [
+	[ 'core/query-pagination-previous' ],
+	[ 'core/query-pagination-numbers' ],
+	[ 'core/query-pagination-next' ],
```

Comment:
These blocks are meant to be changed once the rest of the `comments-pagination` blocks are created.